tests/unit/acquisition/test_optimizer.py

```diff
@@ -802,7 +802,7 @@ def target_function(x: TensorType) -> TensorType:  # [N,V,D] -> [N, V]
         return tf.concat(individual_func, axis=-1)  # vectorize by repeating same function
 
     optimizer = batchify_vectorize(
-        generate_continuous_optimizer(num_initial_samples=1_000, num_optimization_runs=10),
+        generate_continuous_optimizer(num_initial_samples=20_000, num_optimization_runs=10),
```

Comment:
This started failing for Ackley5 following the change. The code itself is working as intended: previously it was generating 1000 initial points and duplicating them 5 times, whereas now it's generating 5000 distinct points. However, the found maximizer is no longer good enough. 

Increasing the number of initial points (significantly) fixes things, but there may be a better way.